python/ray/worker.py

```diff
@@ -1574,8 +1582,16 @@ def wait(object_ids, num_returns=1, timeout=None):
 
     Args:
         object_ids (List[ObjectID]): List of object IDs for objects that may or
-            may not be ready. Note that these IDs must be unique.
-        num_returns (int): The number of object IDs that should be returned.
+            may not be ready. Note that these IDs must be unique. Otherwise, an
+            exception is thrown
+        num_returns (int): The number of object IDs that should be returned.  
```

Comment:
This can be a bit confusing, so here's what I'd suggest. For `num_returns`, document it as

*The maximum number of ready object IDs that should be returned. This value must be at least 1. If it is greater than len(object_ids), it is replaced with len(object_ids). If no timeout is specified, exactly num_returns ready object IDs will be returned. If a timeout is specified, and there are not yet num_returns ready object IDs, then all of the ready object IDs will be returned.*

Reply:
Changed.

Reply:
Fixed linter error.